lib/src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,8 @@
         <activity
             android:name="com.auth0.android.provider.RedirectActivity"
             tools:node="replace">
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
+            <intent-filter android:autoVerify="true">
```

Comment:
Even if inherited from the SDK declaration of this same activity, I rather am sure I'm setting it to true.